samples/v1.2/Elements/TextBlock.Italic.json

```diff
@@ -1,13 +0,0 @@
-{
-	"type": "AdaptiveCard",
```

Comment:
Please delete the corresponding test files as well. (From source\uwp\AdaptiveCardTestApp\Expected:
del /S *TextBlock.Italic*)

Reply:
There's also a copy of the cards in source\uwp\AdaptiveCardTestApp\Expected\SourceCards

---
In reply to: [287427072](https://github.com/microsoft/AdaptiveCards/pull/2970#discussion_r287427072) [](ancestors = 287427072)